apps/src/javalab/JavabuilderConnection.js

```diff
@@ -152,6 +152,9 @@ export default class JavabuilderConnection {
       case WebSocketMessageType.SYSTEM_OUT:
         this.onOutputMessage(data.value);
         break;
+      case WebSocketMessageType.TEST_RESULT:
```

Comment:
since this has the same behavior as system out, we can instead this to avoid duplicate code:
```  
case WebSocketMessageType.SYSTEM_OUT:
case WebSocketMessageType.TEST_RESULT:
        this.onOutputMessage(data.value);
        break;
```